src/onegov/api/token.py

```diff
@@ -1,5 +1,6 @@
 from base64 import b64decode
-from datetime import timedelta
+from datetime import timedelta, timezone
+import datetime
```

Comment:
why not just add datetime to the imports above?

Reply:
Yes. Done